database/src/test/scala/za/co/absa/atum/database/runs/GetPartitioningMainFlowIntegrationTests.scala

```diff
@@ -127,6 +127,8 @@ class GetPartitioningMainFlowIntegrationTests extends DBTestSuite {
       assert(row.getLong("id_flow").contains(1L))
       assert(row.getString("flow_name").contains("Flow1"))
       assert(row.getString("created_by").contains("Sirius"))
+      print(row.getString("flow_name"))
```

Comment:
clean this up pls

Reply:
Thank you, I missed these println. Will remove